apps/main/src/dex/components/PagePoolList/components/TableRowMobile.tsx

```diff
@@ -73,7 +73,7 @@ const TableRowMobile = ({
       }
     }
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [showDetail, sortBy])
+  }, [showDetail, sortBy, volume?.value, tvl?.value, rewardsApy, poolData])
```

Comment:
`// eslint-disable-next-line react-hooks/exhaustive-deps`
strikes again! can't wait to redo this shit in the new MUI datatable